Service-level agreements (#2452)
SoftwareAG/c8y-docs: content/service-terms/service-level-bundle/platform-sla.md

---
title: Platform service-level agreement
layout: bundle
weight: 10
aliases:
  - /platform-sla/
---

> **This Agreement** is made between {{< company-c8y >}} ("Provider") and the Customer ("Customer") who utilizes {{< product-c8y-iot >}} Platform ("Service") for managing Internet of Things ("IoT") devices ("IoT devices", "devices") on Provider's cloud instances ("software-as-a-service", "SaaS").

Reviewer: ```suggestion
This agreement is made between {{< company-c8y >}} ("Provider") and the Customer ("Customer") who utilizes {{< product-c8y-iot >}} Platform ("Service") for managing Internet of Things ("IoT") devices ("IoT devices", "devices") on Provider's cloud instances ("software-as-a-service", "SaaS").
```

### Service description

{{< product-c8y-iot >}} is a comprehensive Internet of Things (IoT) platform designed to enable seamless connectivity, management, analysis and control of IoT devices. This agreement defines the service level of {{< product-c8y-iot >}} Software-as-a-Service operated by {{< company-c8y >}}. {{< product-c8y-iot >}} Software-Operated-as-a-Service and {{< product-c8y-iot >}} Edge are outside the scope of this agreement.

> The agreement applies solely to the base platform excluding optional features or feature not in general availability. Optional features outside the base platform may require a separate service-level agreement. For custom applications developed using the Microservice Hosting functionality, please refer to the [Microservices hosting service-level agreement](/microservice-sdk/microservices-sla/).

Reviewer: ```suggestion
The agreement applies solely to the base platform excluding optional features or features not in general availability. Optional features outside the base platform may require a separate service-level agreement. For custom applications developed using the Microservice Hosting functionality, refer to the [Microservices hosting service-level agreement](/microservice-sdk/microservices-sla/).
```

    * Regular backups are maintained with a 30-day retention period, a Recovery Time Objective (RTO) of 12 hours, and a Recovery Point Objective (RPO) of 24 hours. Backups are stored in the same region where the service is hosted.
    * {{< company-c8y >}} conducts regular drills to validate its disaster recovery procedures.
* **API compatibility management:** The platform maintains compatibility of APIs as outlined in its[ Compatibility policy](/concepts/compatibility-policy/). Transport protocols are managed in accordance with this policy, ensuring consistent and reliable API interactions.
* **Data ownership and portability:** As the data processor, {{< product-c8y-iot >}} ensures that Customer retains full ownership of their data. Customer can export their data at any time using the provided APIs, ensuring control and flexibility over their information.
* **Continuous maintenance and upgrades:** The platform undergoes regular maintenance and upgrades to ensure optimal performance and security. These upgrades happen transparently and without involving Customer.

> For customers with legacy service agreements, these features may differ in accordance with the terms of their original agreements.

Reviewer: ```suggestion
For Customers with legacy service agreements, these features may differ in accordance with the terms of their original agreements.
```

### Customer responsibilities

> As a Customer of the {{< product-c8y-iot >}} platform, we request your acknowledgment of the following responsibilities to ensure the continued security, performance, and efficiency of the service:

Reviewer: ```suggestion
As a Customer of the {{< product-c8y-iot >}} platform, we request your acknowledgment of the following responsibilities to ensure the continued security, performance, and efficiency of the service.
```

* **Library updates:** Customers utilizing {{< company-c8y >}}-provided libraries to build their own applications are responsible for ensuring these libraries are kept up to date with respect to security. In the event of a security vulnerability or other critical update, it is Customer's responsibility to implement the necessary updates in their applications to maintain the integrity and security of their systems. {{< company-c8y >}} cannot be held liable for any security issues arising from outdated libraries in Customer applications.

##### **Capacity management**

* **Scalability considerations:** While the {{< product-c8y-iot >}} platform is designed to be scalable, it may not be able to accommodate sudden, extreme capacity demands (for example, all devices attempting to connect simultaneously after a connectivity outage, or all devices being upgraded at the same time). Customer acknowledges that such requests may be delayed or declined by the platform to maintain overall service stability. Customer is advised to implement an [exponential backoff strategy](https://en.wikipedia.org/wiki/Exponential_backoff).

> * **Soft quotas:** Customer acknowledges the existence of "soft quotas" as documented in the {{< product-c8y-iot >}} documentation [here](/concepts/limits/). These quotas are not strictly enforced but exceeding them may lead to a reduced service level, and Customer is advised to operate within these guidelines.

Reviewer: ```suggestion
* **Soft quotas:** Customer acknowledges the existence of "soft quotas" as documented in [Service quotas](/service-terms/quotas/). These quotas are not strictly enforced but exceeding them may lead to a reduced service level, and Customer is advised to operate within these guidelines.
```

* **Data retention management:** Data storage is included as part of Customer’s subscription. Customer is responsible for configuring appropriate data retention rules within the {{< product-c8y-iot >}} Administration application, balancing their specific use case requirements with budgetary considerations.

> For details on non-permitted uses of {{< product-c8y-iot >}}, please refer to the {{< product-c8y-iot >}} Terms of Service.

Reviewer: ```suggestion
For details on non-permitted uses of {{< product-c8y-iot >}}, refer to the {{< product-c8y-iot >}} Terms of Service.
```

Reviewer: Can we add a link to it?

Author: Good catch, I need to check what the content of the SAG document was.

Author: I sent Jürgen a question what happens with contents of that document. There's even more material in it, it also has data processing. This might be another pull request -- I would park linking until that PR. It was good that you noticed.

### Limitations and constraints

In the interest of transparency and to ensure a mutual understanding of the service capabilities, we kindly ask Customer to acknowledge the following limitations and constraints of the {{< product-c8y-iot >}} platform:

> * **Hard quotas:** Customer acknowledges the existence of hard quotas as detailed in the {{< product-c8y-iot >}} documentation [here](/concepts/limits/). These quotas define maximum thresholds that the platform can support and are essential for maintaining overall system stability.

Reviewer: ```suggestion
* **Hard quotas:** Customer acknowledges the existence of hard quotas as detailed in [Service quotas](/service-terms/quotas/). These quotas define maximum thresholds that the platform can support and are essential for maintaining overall system stability.
```

* **Shared environment considerations:** Customers not utilizing {{< product-c8y-iot >}} Dedicated plans should be aware that their tenant is hosted within a shared environment. As a result, response times may occasionally vary due to shared resource usage, and Customer acknowledges such variations. Furthermore, infrastructure-level information such as HTTP or MQTT access logs cannot be shared with Customer.
* **Data retention and storage costs:** Customer acknowledges that reducing data retention periods does not immediately lead to the reclamation of storage space or a reduction in storage costs due to technical processing requirements.

> * **Distributed IoT system:** Customer acknowledges that IoT systems, by nature, are distributed and Internet-based:

Reviewer: ```suggestion
* **Distributed IoT system:** Customer acknowledges that IoT systems, by nature, are distributed and internet-based:
```

1. Due to acts or conditions outside of Provider’s reasonable control, including, but not limited
to, a Force Majeure event as defined in the agreement above;
2. Initiated by Provider to protect the services or Customer data from unauthorized
access or loss;
3. Caused by Customer’s use of services, hardware, or software not provided by Provider
which affect the availability of the service; or
4. Caused by your use of services other than expressly authorized by, and in accordance
with, the terms of the Agreement and the order form or Customer’s use of the services after we advised you to modify your use of the service, if you did not modify your use as advised.

#### Exclusive remedy

Except as expressly set out in the Agreement, Customer acknowledges and agrees that
Provider’s sole obligation and Customer’s exclusive remedy for Provider’s failure to meet the
service availability requirements are set forth in this service credit commitment.

### Support and maintenance

#### **Support**

> * **Customer support:** Support is provided in accordance with Customer’s selected support plan (Starter, Standard, or Enterprise Active Support), as detailed in a separate support agreement.

Reviewer: add link to Support sla?
